pyproject.toml

```diff
@@ -30,7 +31,6 @@ dependencies = [
     "psutil>=5.9.4",
     "pyansys-tools-versioning>=0.3.3",
     "pyiges[full]>=0.3.1", # Since v0.3.0, the 'full' flag is needed in order to install 'geomdl'
-    "pyvista>=0.38.1",
```

Comment:
I expect to keep the package up to date with latest PyVista version as much as possible. It's probably going to be easier from now on since we are funneling most of PyVista features into a single package. Do you foresee any issues with this?

Reply:
I dont. Just wondering. I also think that going 1-2 patches behind Pyvista allow us to be more stable.